lib/src/Uncertainty/Algorithm/Sensitivity/openturns/SobolIndicesAlgorithmImplementation.hxx

```diff
@@ -163,7 +163,7 @@ protected:
   /** Simulation size */
   UnsignedInteger size_;
 
-  /** Number of Bootstrap sampling */
+  /** Number of Bootstrap sampling size */
   UnsignedInteger nrBootstrap_;
```

Comment:
Maybe we could have the same attribute name:
DistributionFactoryImplementation.hxx:93:  UnsignedInteger bootstrapSize_;